geniza/corpus/templatetags/corpus_extras.py

```diff
@@ -44,3 +49,51 @@ def querystring_replace(context, **kwargs):
         querystring[key] = val
     # return urlencoded query string
     return querystring.urlencode()
+
+
+@register.filter
+def unquote(url):
+    """Template filter to parse URL-encoded URLs.
+
+    Example use::
+        <a href="{{ url|unquote }}">
+    """
+
+    return parse.unquote(url)
```

Comment:
why do we need this? are urls encoded incorrectly?

Reply:
@rlskoeser Huh, I was having this problem with index card URLs but I think it was a result of the outdated data I was working with at the time, and I can no longer replicate. So I'm removing this. Thanks for flagging!

Reply:
oh, yes, that's entirely possible —  we weren't encoding them properly at first, which Rachel caught in testing — when she tried to edit one, django admin validation reported that it wasn't a valid url! Glad we can remove this

Reply:
Done in 4636049 👍